modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/ionchannelmechanismrelease/v0.1.2.json

```diff
@@ -0,0 +1,44 @@
+{
+  "@context": [
```

Comment:
This one is new. Can you provide a description of it.

Reply:
I guess it is a mod file.

Reply:
Yes, it is a collection of mod files.

Reply:
Okay. Then I guess there is a need for an ionchannelmechanism schema.